src/main/java/edu/greenblitz/robotName/Robot.java

```diff
@@ -43,10 +46,9 @@ public void robotInit() {
         Pathfinding.setPathfinder(new LocalADStar());
         CommandScheduler.getInstance().enable();
         initializeLogger();
-        Battery.getInstance().setDefaultCommand(new BatteryLimiter());
         SwerveChassis.init();
-        SwerveChassis.getInstance().resetAllEncoders();
-
+        SwerveChassis.getInstance().setDefaultCommand(new MoveByJoysticks(MoveByJoysticks.DriveMode.NORMAL));
```

Comment:
run a test where you tell the motor to rotate the wheel a full rotation and check if he rotates to there